styles/layouts/_main-navigation.scss

```diff
@@ -44,15 +42,21 @@
 	}
 }
 
+.has-spaced-navbar-fixed-top {
+	padding-top: 72px !important;
+}
+
 .navbar {
 	background-color: inherit;
 	backdrop-filter: blur(20px);
 	transform: translateZ(0px);
 	transition: 0.3s ease;
 	-webkit-transition: 0.3s ease;
 	z-index: 999 !important;
-
+	font-family: 'Work Sans';
```

Comment:
instead of giving font-family here, shouldn't we be replacing the whole app's font? as we use `work-sans` everywhere in the new design, lmkwyt

Reply:
> instead of giving font-family here, shouldn't we be replacing the whole app's font? as we use `work-sans` everywhere in the new design, lmkwyt

only in new Component/Landing for now

Reply:
> only in new Component/Landing for now

I think so too.

Reply:
okay sounds good